components/LoginPage.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default function LoginPage({ setUsername }) {
+  const [inputUsername, setInputUsername] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const navigate = useNavigate();
+
+  const handleLogin = () => {
+    const validUsernames = [
+      "tickle122",
+      "grumpy19",
+      "happyamy2016",
+      "cooljmessy",
+      "weegembump",
+      "jessjelly",
+    ];
+    if (validUsernames.includes(inputUsername)) {
```

Comment:
potentially just make a dropdown box of users for now, save someone that doesn't know the users when they try and try out your app